### ISSUE TITLE
config-rs/tree/master/examples/simple doesn't compile because std::collections doesn't have Map

### ISSUE THREAD

**Issue by tomazvila** [mehcode/config-rs#221]
# Problem

When trying to build application it fails with error:
```
error[E0432]: unresolved import `std::collections::Map`
 --> src/main.rs:2:5
  |
2 | use std::collections::Map;
  |     ^^^^^^^^^^^^^^^^^^^^^ no `Map` in `collections`
```
# Steps to reproduce

1. Copy code from `config-rs/tree/master/examples/simple`
2. Run `cargo build`
3. Get error

# Suggested solution

Replace `std::collections::Map` with `std::collections::HashMap`

**Comment by matthiasbeyer** [comment]
Thanks for your report! I did think CI runs for examples as well, it seems I missed that it doesn't.

Working on a fix in #222

**Comment by matthiasbeyer** [comment]
:partying_face: :tada:
